app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Edit Review</h2>
+
+<section id="edit-review-#{review.id}">
+  <% flash.each do |name, msg| %>
```

Comment:
Also on the to do list would be to move these Flash messages. They were messing up formatting of the pages which is why they don't live on application.html.erb, but in the quest for DRY code, something else to look into.